components/organisms/accounts/statement/statement.vue

```diff
@@ -63,102 +63,121 @@
     </b-container>
 </template>
 <script lang="ts">
-import { Component, mixins, Prop } from 'nuxt-property-decorator'
+import { defineComponent, computed, ComputedRef, PropType, useRoute } from '@nuxtjs/composition-api'
 import dot from 'dot-object'
 import { DateTime } from 'luxon'
-import BaseMixin from '~/mixins/BaseMixin'
-import RouterMixin from '~/mixins/RouterMixin'
-import FiltersMixin from '~/mixins/FiltersMixin'
+import { useStores } from '~/composables/useStores'
+import { useAccounts } from '~/composables/useAccounts'
+import { useFilters } from '~/composables/useFilters'
+import { useRouterFilter } from '~/composables/useRouterFilter'
 import { GetManagedAccountStatementRequest } from '~/plugins/weavr-multi/api/models/managed-instruments/managed-account/requests/GetManagedAccountStatementRequest'
-import AccountsMixin from '~/mixins/AccountsMixin'
 import { OrderEnum } from '~/plugins/weavr-multi/api/models/common/enums/OrderEnum'
 
-@Component({
+export default defineComponent({
+    name: 'AccountStatementComponent',
+    // TODO: Update this after nuxt bridge
```

Comment:
we can remove this an use setup tag i think. We can then just rename the file name to account statements and refactor imports where necessary. then we can use setup tag.

Reply:
The issue here is the way assets are imported, I think we're blocked by [this](https://weavr-payments.atlassian.net/browse/WEB-176) if not we have to use defineComponent()